app/src/views/Confirm/Confirm.vue

```diff
@@ -470,8 +471,9 @@ export default {
     },
     getCurrencyMultiplier() {
       const { selectedCurrency, currencyData } = this.$store.state || {}
-      let currencyMultiplier = 1
-      if (selectedCurrency !== 'ETH') currencyMultiplier = currencyData[selectedCurrency.toLowerCase()] || 1
+      let currencyMultiplierNum = 1
+      if (selectedCurrency !== 'ETH') currencyMultiplierNum = currencyData[selectedCurrency.toLowerCase()] || 1
+      const currencyMultiplier = new BigNumber(currencyMultiplierNum)
```

Comment:
Can you return this as one liner than breaking in 2?

Reply:
Done